ochazuke/models.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""Models and methods for working with the database."""
+
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy.dialects import postgresql
+
+db = SQLAlchemy()
+
+
+class Milestone(db.Model):
```

Comment:
nit: Documenting with docstring this class and all the subsequent ones.

Reply:
You're absolutely right, sorry! I'll do that asap.